sycl/test-e2e/Regression/no-split-reqd-wg-size.cpp

```diff
@@ -0,0 +1,39 @@
+// This test checks that with -fsycl-device-code-split=off, two kernels
+// with different reqd_work_group_size dimensions can be launched.
+// RUN: %{build} -fsycl -fsycl-device-code-split=off -o %t.out
+// RUN: %{run} %t.out
+#include <sycl/sycl.hpp>
+
+constexpr int WGSIZE = 4;
+
+using namespace sycl;
+
+void kernel_launch_2(queue &q) {
```

Comment:
probably the func should be called `kernel_launch_1` as the dimension is one, and the func below `kernel_launch`, should be called `kernel_launch_2`?